### DISCUSSION TITLE
Is this ILS approach normal/displayed correctly?

### DISCUSSION THREAD

**Discussion by cadilhac** [flybywiresim/aircraft#3586] [Q&A] [answered]
I'm trying to replicate the pattern setup from the Airline2Sim A320 course (they use FSLabs on P3D I guess). They takeoff from Menorca LEMH and setup the FPLN as a simple Rwy 19 departure + ILS19Z arrival. In the course this draws a plain straight line with 2 fixes on the approach axis.
With the A32NX, building this setup produces this FPLN:

![arc](https://user-images.githubusercontent.com/50608/108283108-b1d85800-7150-11eb-8bbb-2a08dd6d8019.png)

Why such an arc? Is it normal? Is it a Neo thing? It does not appear in the FPLN so it's not removable.

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
![Screenshot 2021-02-25 124111](https://user-images.githubusercontent.com/72575144/109193871-cfb44700-7766-11eb-9bf6-0949472f8fe6.jpg)
it looks right to me

**Reply by cadilhac** [reply]
ok. Didn't know it would be displayed like this.

**Comment by RustyCT1** [comment]
The curve it calculates beforehand; the beautiful arcing transitions between waypoints, especially on approach and final.  I have observed many of these in my routes, and now actually plan for them in building flight plans.  I think the traveled route is elegant.

Not to mention the thousands of calculations per second sending instructions to physical actuators, executing thousands of minute corrections in the flight plane and airspeed, so that arrival to a waypoint is at the correct altitude and speed, even if it did not draw a direct path between waypoints.
